### ISSUE TITLE
On-hover dialogues don't disappear on Linux

### ISSUE THREAD

**Issue by Kat-zorn** [modrinth/code#973]
### Describe the bug

On Linux I have the issue that all of the on-hover dialogues don't disappear when I remove my mouse from them.

### Steps to reproduce

1. Open launcher.
2. Hover mouse over a button (such as Home, Browse, Library, Log in/ -out).
3. A little box will pop-up, with one or two words.
4. Move mouse somewhere else.
5. The box will stay.

### Expected behavior

Instead of step 5, the box will go away.

### System information

```uname -a```
Linux blazingblast 6.7.0-rc8-00130-g6d0dc8559c84 #52 SMP PREEMPT Fri Jan  5 20:37:00 CET 2024 x86_64 GNU/Linux
```pacman -Q modrinth-app-bin```
modrinth-app-bin 0.6.3-1

### Additional context

![image](https://github.com/modrinth/theseus/assets/69185984/3c8c33d9-b342-4bff-88a2-bacc10a1d4ce)

**Comment by triphora** [comment]
I can reproduce this, but it seems to only occur when it's active while navigating to another page.

**Comment by tobeqz** [comment]
This happens to me almost every time (but not always) in the AUR package `modrinth-app` and when building the project in release mode, but for some reason I can't reproduce it when I'm in `pnpm tauri dev`. Maybe something weird is going on with webkit2gtk?

**Comment by tobeqz** [comment]
I compiled the release version with debug assertions enabled. I can reproduce this bug by opening the app and immediately clicking one of my profiles. It seems like an error is being thrown somewhere in `popper.js` internals which causes this to happen.
![image](https://github.com/modrinth/theseus/assets/83136264/f39859de-1191-46f1-b9f1-67fbb7ef65d6)
![image](https://github.com/modrinth/theseus/assets/83136264/f53aa484-8a16-4ef4-8440-24b7a7c60220)

I tried to set a breakpoint on uncaught exceptions to figure out what exactly is being passed into `Reflect.get` (my guess is `undefined`), but this bug is actual nightmare fuel and I cannot reproduce it while DevTools are open.

Reloading the page does work as a workaround, but unfortunately, AFAIK there is no way to reload the page in the release builds available in the AUR. I think it would be nice to give users the option to reload for situations like this and if it's wanted I can add a hotkey like Ctrl+R or F5.
